### DISCUSSION TITLE
Nextion LCD compatibility

### DISCUSSION THREAD

**Discussion by DCullenV** [Zer0-bit/gaggiuino#215] [Q&A] [answered]
The [2.4" Nextion LCD](https://bit.ly/3CAUzPj) is listed for base functionality of the Gaggiuino. Can I use the larger 2.8" or 3.5" Nextion LCDs also listed on the same AliExpress page? Is there a modification needed to do this?

Note:

- The 2.8" Nextion LCD has the same resolution as the 2.4" (320x240).
- The 3.5" Nextion LCD has a higher resolution (480x320).

**Comment by Zer0-bit** [comment, marked as answer]
You can use any other nextion lcd rocking the same resolution, other versions won't scale.

**Reply by jvle** [reply]
Can I use the existing nexton-lcd.HMI, change device to 3.5 nextion display, the move the buttons around?  Would that work without changing Arduino code?

**Comment by showmak** [comment]
Then you have to modify the housing I guess if you want to use a different size of LCD .
